payu/models/cice.py

```diff
@@ -105,7 +141,8 @@ def get_prior_restart_files(self):
         return [sorted(restart_files)[-1]]
 
     def get_ptr_restart_dir(self):
-        return self.ice_in['setup_nml']['restart_dir']
+        # return self.ice_in['setup_nml']['restart_dir']
+        return os.path.relpath(self.work_input_path, self.work_path)
```

Comment:
Should we use absolute path here?  Any advantage/disadvantage?

I've generally tried to use abspaths in the path, though not for any explicit reason.

Reply:
It is used to print the path to the restart file in the `ice.restart_file` pointer file. I don't really know how CICE groks this, though I guess a full path would work. As it is for a pretty specific purpose I think I'd rather leave that for now.